PocketKit/Sources/PocketKit/MyList/ReaderActionsWebActivity/ReaderActionsWebActivity.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+import Textile
+
+enum WebActivityTitles: String {
```

Comment:
```suggestion
enum WebActivityTitle: String {
```

Realistically, a case is only ever a single type of title, so let's reflect that in the type name.

Reply:
done